__init__.py

```diff
@@ -117,71 +104,62 @@ def __init__(self):
         super().__init__('Mark2')
 
         self.settings['auto_brightness'] = False
-        self.settings['use_listening_beep'] = False
+        self.settings['use_listening_beep'] = True
```

Comment:
This was hanged because of the project rollover feedback. Waiting until the beep before speaking was input we gave to them. I simply changed this to remain consistent. Technically this should come from the master config setting. I just changed the value to match the project rollover solution we sent them.